### PR TITLE
Create index html page

### PR DESCRIPTION
**Pull request by Ibrahimnabeel9**
Relates #3

### CODE REVIEW

**Review by techhana2014 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>landing page</title>
+  </head>
+  <body>
+    <div class="title">
```
Comment (yazan05) [inline index.html:11]: remove div and add article

File: index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      <h2 class="header-title">COVID 19</h2>
+      <p class="header-paragraph">Stay Updated & Stay Safe</p>
+    </div>
+
+    <div class="navigation-btns">
+      <a class="skip-btn" href='statistics/statistics.html'>Skip</a>
+      <a class="start-btn" href='page1.html'>Countinue</a>
+    </div>
+
+    <div class="background">
```
Comment (yazan05) [inline index.html:21]: replace div with figure

**Review by yazan05 — CHANGES_REQUESTED** [review]
<a class="start-btn" href='page1.html'>Countinue</a> >>>> edit href  : instructions.html

File: index.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Sans+KR:wght@400;500;700&display=swap" rel="stylesheeet">
+    <title>landing page</title>
+  </head>
+  <body>
+    <article class="title">
+      <h2 class="header-title">COVID 19</h2>
+      <p class="header-paragraph">Stay Updated & Stay Safe</p>
+    </article>
```
Comment (Ibrahim-Jarada) [inline index.html:15]: After you enter the body It better to use `<section> ` not `<article> `

File: index.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+    <section class="navigation-btns">
+      <a class="skip-btn" onclick="page.href='tasks/tasks.html'">Skip</a>
+      <a class="start-btn" href="#">Countinue</a>
+    </section>
```
Comment (Ibrahim-Jarada) [inline index.html:20]: Use `<nav>` instead of `<section>` for your navigation  bar

**Review by Ibrahim-Jarada — CHANGES_REQUESTED** [review]
Always use semantic HTML
Check this link [Semantic HTML](https://github.com/gazaskygeeks/Coding-Foundations-course/blob/master/coursebook/Week%2001/session-02/semantic.md)

**Review by yazan05 — APPROVED** [review]

**Review by Ibrahim-Jarada — APPROVED** [review]

**Pull request merged**